IPython/core/latex_symbols.py

```diff
@@ -1294,4 +1294,5 @@
     "\\mttseven" : "𝟽",
     "\\mtteight" : "𝟾",
     "\\mttnine" : "𝟿",
+    "\\snowman" : "☃",
```

Comment:
If I understand correctly, the unicode_name_matches means this doesn't need to be added here?